common/src/main/java/com/github/teamfusion/summonerscrolls/common/util/cooldowns/SummonerItemCooldowns.java

```diff
@@ -0,0 +1,77 @@
+package com.github.teamfusion.summonerscrolls.common.util.cooldowns;
+
+import com.google.common.collect.Maps;
+import java.util.Iterator;
+import java.util.Map;
+import net.minecraft.util.Mth;
+import net.minecraft.world.item.ItemStack;
+
+
+public class SummonerItemCooldowns {
+    private final Map<ItemStack, SummonerCooldownInstance> cooldowns = Maps.newHashMap();
+    private int tickCount;
+
+    public SummonerItemCooldowns() {
+    }
+
+    public static SummonerItemCooldowns createSummonerItemCooldowns(){
+        return new SummonerItemCooldowns();
+    }
```

Comment:
Yeah, no. Just call the constructor instead of making a useless static method.
Also, don't create empty constructors, they aren't needed and provide no useful information to the developer.

Note: If there is an actual reason for this, i.e. mixins fucking with the `new` statement (which they don't), state so in a comment.

Reply:
Okay, but add a TODO comment, so you don't forget to refactor this class to be more in-line with our use case